### PR TITLE
[#1] added create calendar functionality

### PR DESCRIPTION
**Pull request by Syuziko**

### CODE REVIEW

**Comment by Biacode** [comment]
Merge master branch

File: src/main/java/org/biacode/jcronofy/api/client/CronofyClient.java [hunk 1]
```diff
@@ -43,6 +43,13 @@
      */
     CronofyResponse<ListCalendarsResponse> listCalendars(final ListCalendarsRequest request);
 
+    /**
+     * Create calendar
```
Comment (Biacode) [inline src/main/java/org/biacode/jcronofy/api/client/CronofyClient.java:47]: Add space

File: src/main/java/org/biacode/jcronofy/api/client/impl/CronofyClientImpl.java [hunk 2]
```diff
@@ -174,6 +174,23 @@ public CronofyClientImpl(final Client client) {
         }
     }
 
+    @Override
+    public CronofyResponse<CreateCalendarResponse> createCalendar(final CreateCalendarRequest request) {
```
Comment (Biacode) [inline src/main/java/org/biacode/jcronofy/api/client/impl/CronofyClientImpl.java:178]: Tested in real environment?

File: src/main/java/org/biacode/jcronofy/api/client/impl/CronofyClientImpl.java [hunk 3]
```diff
@@ -174,6 +174,23 @@ public CronofyClientImpl(final Client client) {
...
+        final CronofyResponse<CreateCalendarResponse> response = new CronofyResponse<>();
+        final Response result = getClient()
+                .target(BASE_PATH)
+                .path(API_VERSION)
+                .path(CALENDARS_PATH)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .header(AUTH_HEADER_KEY, getAccessTokenFromRequest(request))
+                .post(Entity.entity(request, MediaType.APPLICATION_JSON_TYPE)
+                );
+        final int statusCode = result.getStatus();
```
Comment (Biacode) [inline src/main/java/org/biacode/jcronofy/api/client/impl/CronofyClientImpl.java]: use less

**Review by Biacode — COMMENTED** [review]
Good job! :)

File: src/main/java/org/biacode/jcronofy/api/client/impl/CronofyClientImpl.java (hunk 2 above)
Comment (Syuziko) [reply src/main/java/org/biacode/jcronofy/api/client/impl/CronofyClientImpl.java:178]: Yep

**Pull request merged**